src/main/java/core/basesyntax/WorkWithFile.java

```
@@ -1,7 +1,65 @@
 package core.basesyntax;
 
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.Files;
+
 public class WorkWithFile {
+    private static final String SUPPLY_WORD = "supply";
+    private static final String BUY_WORD = "buy";
+
     public void getStatistic(String fromFileName, String toFileName) {
 
+        String fileContent = readFromFile(fromFileName);
+        String[] lines = fileContent.split("\n");
+        int supplySum = 0;
+        int buySum = 0;
+        int result;
+        for (String line : lines) {
+            String[] splitedLine = line.split(",");
+            int value = Integer.parseInt(splitedLine[1].trim());
```

Parsing the amount with `Integer.parseInt(splitedLine[1].trim())` may throw `NumberFormatException`. The checklist says not to ignore exceptions and to rethrow a RuntimeException with the original exception as the cause. Quote: "Don't ignore exceptions ... Better re-throw `RuntimeException` with the original exception and some message in the parameters". Wrap parsing in try-catch and rethrow with informative message and cause.